include/vrv/object.h

```diff
@@ -471,6 +471,10 @@ class Object : public BoundingBox {
     void StableSort(Compare comp) { std::stable_sort(m_children.begin(), m_children.end(), comp); }
 
     virtual void ReorderByXPos();
+
+    Object *FindNextChildOfType(Comparison *comp, Object *start);
```

Comment:
This just takes a comparison and seems to be generic to any comparison. If this is the case, rename to something like `FindNextChildByComparison`. Otherwise, take a `ClassId` argument.